Turn/include/Store.h

```diff
@@ -6,6 +6,15 @@
 class Store {
     public:
         void StoreFront(Player*);
+        void SellingItems(Player*);//impliments selling of items
+        int  getJeremysCoin(){
```

Comment:
Only include function declarations in header files, put the implementation in the source files